src/config/materials.js

```diff
@@ -123,6 +123,19 @@ export default {
 
   filters: {
     fields: {
+      material: {
+        field: 'collectionobjects_common:materialGroupList.material',
```

Comment:
Well, that's annoying. I guess we'll have to live with it for now.

Reply:
I guess there's also the option of switching to `collectionspace_deurn:materialGroupList` for the material tenant as well. I didn't see it as being necessary at the time, but might be nice to have the es indexes be consistent.

Reply:
That would be fine. I don't feel strongly either way. In `DefaultESDocumentWriter.denormMaterialFields` you'd have to account for the `material` field possibly being a ref name, since it is one in the materials tenant, and do a conditional de-urning.

Reply:
I made the changes to trim out the redundant messages and with this only overriding the `field` it's not so bad. It seems like there are a few places where other tenants override fields as well so I'm ok with it.